proposals/csharp-10.0/GlobalUsingDirective.md

```diff
@@ -1,5 +1,7 @@
 # Global Using Directive
 
+[!INCLUDE[Specletdisclaimer](~/docs/csharp/includes/speclet-disclaimer.md.md)]
```

Comment:
`.md.md`?

Reply:
That's my mistake. Thanks for catching it.